### DISCUSSION TITLE
Need to open an lfs file - noob

### DISCUSSION THREAD

**Discussion by solveitservices** [git-lfs/git-lfs#5908] [Q&A]
I'm not a developer, but do work in tech.

One of my clients received some documets like: "document-1.pdf.lfs" and asked me how they can open it.

So far, I've installed Git and Git LFS following the basic instructions here: https://git-lfs.com/ and run the command: git lfs install

This was successful.

I don't quite understand what to do next. Step to says "In each Git repository..."

So I suppose I need a Git repository. Is this referring to some local directory that I designate as a git reposity? I opened the Git Gui, and selected the option to Create New Repository. Is this correct so far?

The instructions continue: In each Git repository where you want to use Git LFS, select the file types you'd like Git LFS to manage (or directly edit your .gitattributes). You can configure additional file extensions at anytime.

example: git lfs track "*.psd"

So on my machine, should I just browse via the command line to the repostiory I created and run the command: git lfs track "*.pdf" ?

Then the command: git add .gitattributes

I've done this, but not sure what to do next. The instructions say: There is no step three. Just commit and push to GitHub as you normally would; for instance, if your current branch is named main:
git add file.psd
git commit -m "Add design file"
git push origin main

I don't understand what this is referring to, or how it relates to where these .lfs files my client has received should reside, and how I go about opening them.

Can someone point me in the right direction?

Thank you.

**Comment by chrisd8088** [comment]
Hey, thanks for the questions, and I hope we can be helpful.

The first thing I'd check is what the file format of the files actually is.  You mention that they have names like `document-1.pdf.lfs`.  I'm not sure that that implies that they're managed with Git and Git LFS at all.

As a brief summary of how the Git LFS client works, it's an extension to the Git source code management tool, which helps users of Git to manage certain files (usually large or binary ones) by storing them outside of their Git projects.  But neither Git nor Git LFS requires a user to rename files, or change their extensions.  In particular, the `.lfs` file extension is not something that's required by or even suggested for Git LFS users; in fact, quite the opposite, because we usually want and expect users to retain their original file names.  And Git LFS isn't a file format at all; it's a way of storing any kind of file, so we don't need or want users to change their filenames.

So if you have some files with a `.lfs` extension, I suspect that means Git LFS isn't involved at all.  At a guess, it's more likely to be a custom file format.

For instance, there appears to be an LFS file compression format which was used for certain games.  I found a couple of links to some resources for this particular format:
- https://steamcommunity.com/app/254700/discussions/0/558749191351715121/
- https://github.com/emoose/re4-research

But there could be other file formats which happened to use that extension, or maybe it's something unique to the source environment for the files.  Or perhaps it's just a random extension (i.e., maybe the files are just PDFs, and someone put an odd extra file extension on their names?)

Regardless, I'd look for some tools appropriate to your operating system (Windows, macOS, or Linux) which could probe the file contents and give you some hints.  On Linux and macOS, for example, the `file(1)` command can sometimes help, just by running a command like `file document-1.pdf.lfs`.
